contents/docs/product-analytics/autocapture.mdx

```diff
@@ -23,6 +23,182 @@ Autocapture is available in the following SDKs:
 - [React Native](/docs/libraries/react-native) - disabled by default
 - [iOS](/docs/libraries/ios) - disabled by default
 
+<CalloutBox icon="IconInfo" title="Looking for session and exception autocapture?" type="fyi">
+
+  You can also autocapture session and exception events. See [session replay](/docs/session-replay/installation) and [error tracking](/docs/error-tracking/start-here) for more information.
+
+</CalloutBox>
+
+## Events captured by autocapture
+
+PostHog automatically captures the following events when autocapture is enabled:
+
+import Tab from "components/Tab"
+
+<Tab.Group tabs={['Web', 'iOS', 'React Native']}>
+<Tab.List>
+<Tab>Web</Tab>
+<Tab>iOS</Tab>
+<Tab>React Native</Tab>
+</Tab.List>
+<Tab.Panels>
+<Tab.Panel>
+
+| Event | Description |
+|-------|-------------|
+| `$autocapture` | On interactions with: `a`, `button`, `form`, `input`, `select`, `textarea`, `label`, and elements with `contenteditable="true"`. Autocaptures are [displayed](https://app.posthog.com/activity/explore) with names like `clicked span with text "Delete"`, but filtering for **Autocapture** will show you all autocaptured events. |
+| `$pageview` | When the user navigates to a new page. |
+| `$pageleave` | When the user navigates away from a page. |
+| `$rageclick` | A second click within 1000ms within 30px of the first click. |
+| `$copy_autocapture` | Clipboard events. Disabled by default. See [clipboard autocapture](#clipboard-autocapture). |
```

Comment:
Hmm, usually when we refer to the `autocapture` feature we are not talking about the 3 middle ones, only about the actual `$autocapture` events. Do you think it's preferrable to include these here?

Might need to update our wording in other places